README.md

```diff
@@ -53,6 +53,8 @@ The service requires the following configuration parameters:
 
 - <a id="properties/access_grant_max_days"></a>**`access_grant_max_days`** *(integer)*: The maximum number of days that the access can be granted. Default: `730`.
 
+- <a id="properties/access_grant_max_extend"></a>**`access_grant_max_extend`** *(integer)*: Max factor by which a data steward may extend the access grant. Default: `5`.
```

Comment:
This is difficult to understand as is. We should specify that it is a factor to multiply access_grant_max_days.

Reply:
Ok, I improved the description a bit.